packages/shared/src/aws/tile.metadata.tileset.ts

```diff
@@ -137,6 +140,14 @@ export class TileMetadataTileSet extends TaggedTileMetadata<TileMetadataSetRecor
         return `ts_${record.name}_${record.projection}_${tag}`;
     }
 
+    idSplit(record: TileMetadataSetRecord): { name: string; projection: string; tag: string } | null {
+        const [prefix, name, projection, tag] = record.id.split('_');
+        if (prefix != 'ts') return null;
+        if (!Epsg.parse(projection)) return null;
+        if (!parseMetadataTag(tag)) return null;
+        return { name, projection, tag };
```

Comment:
do we also want to parse `v000006` into `{ name: string, projection: .., version: number (6)}`

Reply:
I believe for the vacuum imagery script, we only need the production metadata. However, it seem good to parse versions here as well. I will make the changes.